### DISCUSSION TITLE
Welcome to router Discussions!

### DISCUSSION THREAD

**Discussion by posva** [vuejs/router#1604] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jd-solanki** [comment]
Hi 👋🏻

I am JD. I am frontend developer at [ThemeSelection](https://themeselection.com/) and working on open source utility first Vue component library [Anu](https://github.com/jd-solanki/anu).

For routing in ThemeSelection's admin panels we are using [vite-plugin-pages](https://github.com/hannoeru/vite-plugin-pages) but it isn't officially supported by vue-router so I wanted to ask if there are any plans to develop and merge [unplugin-vue-router](https://github.com/posva/unplugin-vue-router)? Is it stable for production use?
